RetrospectiveExtension.Frontend/webpack.config.js

```diff
@@ -8,11 +8,18 @@ const BUILD_DIR = path.resolve(__dirname, 'dist');
 const APP_DIR = path.resolve(__dirname, '');
 
 module.exports = {
-  devtool: 'source-map',
```

Comment:
My guess is that if we pack the extension in production mode, the devServer will not be picked up. Can you verify this. Also this change can change the behavior of the production extension. @polatengin  Do you have any experience with this.
I am lenient towards the safer route and leaving this file unchanged until we verify that it is safe